docs/content/direct/architecture.md

```diff
@@ -102,11 +102,11 @@ in the [KubeFlex Architecture](https://github.com/kubestellar/kubeflex/blob/main
 There are currently two roles for spaces managed by KubeFlex: Inventory and Transport Space 
 (ITS) and Workload Description Space (WDS). The former runs the [OCM Cluster Manager](#ocm-cluster-manager) on a vcluster-type control plane, and the latter runs on a k8s-type control plane.
 
-An ITS holds the OCM inventory (`ManagedCluster`) objects and mailbox namespaces. The mailbox namespaces and their contents are implementation details that users do not deal with. Each mailbox namespace corresponds 1:1 with a WEC and holds `ManifestWork` objects managed by the central KubeStellar controllers.
+An ITS holds the inventory and the mailbox namespaces. The inventory is anchored by `ManagedCluster` objects that describe the WECs. For each WEC there may also be a `ConfigMap` object (in the `customization-properties` namespace) that carries additional properties of that WEC; this `ConfigMap` is used in cutomizing the workload to the WEC. The mailbox namespaces and their contents are transport implementation details that users do not deal with. Each mailbox namespace corresponds 1:1 with a WEC and holds `ManifestWork` objects managed by the central KubeStellar controllers.
 
 A WDS holds user workload objects and the user's objects that form the interface to KubeStellar control. 
 Currently, the user control objects are `BindingPolicy` and `Binding` objects.
-We plan to later add objects to specify customization and summarization.
+Future development may define more kinds of control objects that exist in the WDS.
 
```

Comment:
Perhaps better to say "hosted in the WDS" instead than "that exist in the WDS"

Reply:
done